OgreMain/include/OgreSceneLoaderManager.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __SCENELOADERMANAGER_H__
+#define __SCENELOADERMANAGER_H__
+
+#include "OgreSingleton.h"
+#include "OgreSceneLoader.h"
+
+namespace Ogre {
+    
+    class _OgreExport SceneLoaderManager : public Singleton<SceneLoaderManager>
+    {
+    public:
+        SceneLoaderManager();
+        virtual ~SceneLoaderManager();
+        void registerSceneLoader(const String& name, SceneLoader *sl);
+        SceneLoader *getByName(const String& name);
```

Comment:
I think it makes more sense to provide a high-level "load" here that picks the right sceneloader or throws otherwise.

Reply:
Still, what about checking available loaders at runtime?

Reply:
Will create seperate method for checks.